data/brands/shop/furniture.json

```diff
@@ -768,6 +768,17 @@
         "shop": "furniture"
       }
     },
+    {
+      "displayName": "Nick Scali",
+      "id": "nickscali-0da980",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "brand": "Nick Scali",
+        "brand:wikidata": "Q17053453",
```

Comment:
This wikidata have a [en wikipedia](https://en.wikipedia.org/wiki/Nick_Scali_Furniture) and english may had been wildly used in Australia, so it's better to add this tag:```"brand:wikipedia":"en:Nick_Scali_Furniture"``` ♥

Reply:
we don't have to worry about these anymore! - since #4716, #4717 now the build_wikidata.js script will fill in and fix the companion `brand:wikipedia` tag in automatically 👍